Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -149,7 +149,8 @@ namespace internal{
  * The normal vector to each face is chosen pointing on the side of the face
  * where its sequence of vertices is seen counterclockwise.
  *
- * @pre `CGAL::is_closed(tm)`
+ * @pre `CGAL::is_closed``(tm)`
+ * @pre \link CGAL::is_closed `CGAL::is_closed(tm)` \endlink
```

Comment:
Why double, or just as test double to see what happens?
Why not also is_triangle_mesh ?

Reply:
Just to see how it looks like with the two approaches.